pkg/kubectl/cmd/create_clusterrole.go

```diff
@@ -133,6 +134,12 @@ func (c *CreateClusterRoleOptions) Validate() error {
 				return fmt.Errorf("invalid verb: '%s' for nonResourceURL", v)
 			}
 		}
+
+		for _, nonResourceURL := range c.NonResourceURLs {
+			if !strings.HasPrefix(nonResourceURL, "/") {
```

Comment:
A single "*" is also allowed nonResourceURL.
But "*" is allowed in the tail, so this is invalid: /foo/*bar